test/src/657-tarball_remove_base_directory/main

```diff
@@ -138,6 +138,37 @@ cvmfs_run_test() {
     echo "*** Directory actually removed $repo_dir/$to_remove_dir"
   fi
 
+  #===========================================================================#
+
+  echo "*** we removed the base directory while ingesting another tar"
+  echo "*** trying to remove files and directory without ingesting anything"
+
+  #===========================================================================#
+
+  echo "*** trying to remove a non existing file"
+  cvmfs_server ingest -d $to_keep_dir/non_existing_file.tar $CVMFS_TEST_REPO || return $?
+
+  echo "*** trying to remove a single file"
+  file=$to_keep_dir/tarball_foo/a/b/c/3.txt
+  cvmfs_server ingest -d $file $CVMFS_TEST_REPO || return $?
+  if [ -f $file ]; then
+    echo "*** Error found file $file"
+    return 10
+  else
+    echo "*** Removed file: $file"
+  fi
+
+  echo "*** trying to remove multiple file"
+  file1=$to_keep_dir/tarball_foo/a/b/c/1.txt
+  file2=$to_keep_dir/tarball_foo/a/b/c/2.txt
+  cvmfs_server ingest -d $file1 -d $file2 $CVMFS_TEST_REPO || return $?
+  if [ -f $file1 ] || [ -f $file2 ]; then
+    echo "*** Error found file $file1 or $file2"
+    return 11
+  else
+    echo "*** Removed files: $file1 and $file2"
+  fi
+
```

Comment:
It would be good to add `cvmfs_server check -i` commands after the various ingestion steps.

Reply:
Done!